### PR TITLE
Remove dependency on blockdiag

### PR DESCRIPTION
**Pull request by p12tic**
funcparserlib requires setuptools older than 58 because it still uses 2to3 functionality which has been removed. This breaks installation of Buildbot on setuptools newer than 58. funcparserlib is required by blockdiag which we use for automatic generation of diagrams in our docs. This PR switches the diagrams to svg and removes the dependency.

### CODE REVIEW

File: master/docs/_images/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-SOURCES = overview_src.svg changes_src.svg workers_src.svg master_src.svg
+SOURCES = overview_src.svg changes_src.svg workers_src.svg master_src.svg multimaster_src.svg auth_rules_src.svg
```
Comment (rjarry) [inline master/docs/_images/Makefile:2]: I did not understand what are these `_src` files for. Could you explain?

File: master/docs/developer/www-data-module.rst [hunk 2]
```diff
@@ -10,12 +10,6 @@ Its main purpose is to handle the 3 way binding.
 2 way binding is the angular MVVM concept, which seamlessly synchronise the view and the model.
 Here, we introduce an additional way of synchronisation, which is from the server to the model.
 
-.. blockdiag::
-
-    blockdiag {
-      View <- Model <- Server;
-    }
-
```
Comment (rjarry) [inline master/docs/developer/www-data-module.rst:18]: You did not replace this with an image?

File: master/docs/developer/www-data-module.rst [hunk 3]
```diff
@@ -44,13 +38,6 @@ For example, the Change wrapper decodes the author name and email from the "auth
 
 Each wrapper class also has specific access methods, which allow to access more data from the REST hierarchy.
 
-.. blockdiag::
-
-    blockdiag {
-      data.getBuilds -> Collection -> Builds -> b.getSteps -> Collection -> Steps;
-    }
-
-
```
Comment (rjarry) [inline master/docs/developer/www-data-module.rst:53]: You did not replace this with an image?

**Review by rjarry — CHANGES_REQUESTED** [review]

File: master/docs/_images/Makefile (hunk 1 above)
Comment (p12tic) [reply master/docs/_images/Makefile:2]: src files are for editing with inkscape. It contains additional information needed for e.g. editing of text. It's later exported to svg which contains text outline that can no longer be edited via inkscape.

File: master/docs/developer/www-data-module.rst (hunk 2 above)
Comment (p12tic) [reply master/docs/developer/www-data-module.rst:18]: Too lazy as explained in the message of the first commit. Let's use svg from your PR here.

File: master/docs/_images/Makefile (hunk 1 above)
Comment (rjarry) [reply master/docs/_images/Makefile:2]: ok thanks

File: master/docs/developer/www-data-module.rst (hunk 2 above)
Comment (rjarry) [reply master/docs/developer/www-data-module.rst:18]: :+1:

**Review by rjarry — APPROVED** [review]
looks good to me :+1:

**Pull request merged**